docs/howto/admin/systemd.rst

```diff
@@ -0,0 +1,77 @@
+.. _howto/admin/systemd:
+
+================================
+Customizing ``systemd`` services
+================================
+
+By default, TLJH configures two ``systemd`` services to run JupyterHub and Traefik.
+
+These services come with a default set of settings, which are specified in
+`jupyterhub.service <https://github.com/jupyterhub/the-littlest-jupyterhub/blob/master/tljh/systemd-units/jupyterhub.service>`_ and
+`traefik.service <https://github.com/jupyterhub/the-littlest-jupyterhub/blob/master/tljh/systemd-units/traefik.service>`_.
+They look like the following:
+
+.. code-block:: bash
+
+  [Unit]
+  Requires=traefik.service
+  After=traefik.service
+
+  [Service]
+  User=root
+  Restart=always
+  WorkingDirectory=/opt/tljh/state
+  PrivateTmp=yes
+  PrivateDevices=yes
+  ProtectKernelTunables=yes
+  ProtectKernelModules=yes
+  Environment=TLJH_INSTALL_PREFIX=/opt/tljh
+  ExecStart=/opt/tljh/hub/bin/python3 -m jupyterhub.app -f jupyterhub_config.py --upgrade-db
+
+  [Install]
+  WantedBy=multi-user.target
+
+
+However in some cases, admins might want to have better control on these settings.
+
+For example when mounting shared volumes over the network using Samba, these namespacing settings might be a bit too strict
+and prevent users from accessing the shared volumes.
+
+
+Overriding settings with ``override.conf``
+==========================================
+
+To override the settings, it is possible to provide a custom ``/etc/systemd/system/jupyterhub.service.d/override.conf`` file
+with the following content:
+
+.. code-block:: bash
+
+    [Service]
+    PrivateTmp=no
+    PrivateDevices=no
+    ProtectKernelTunables=no
+    ProtectKernelModules=no
+
+Then make sure to reload the daemon and the ``jupyterhub`` service:
+
+.. code-block:: bash
+
+    sudo systemctl daemon-reload
+    sudo systemctl restart jupyterhub
+
+Checking the status should look like the following:
```

Comment:
Maybe also provide the command to check the status? I believe it's already specified somewhere in the docs, but I think it's best to have it here also just for completeness.